windows/win_get_url.ps1

```diff
@@ -67,21 +67,31 @@ If ($force -or -not (Test-Path $dest)) {
 }
 Else {
     Try {
-        $webRequest = [System.Net.HttpWebRequest]::Create($url)
-
+        #before download, head to compare -> more efficient
+        $webRequestHead = [System.Net.HttpWebRequest]::Create($url)
         if($username -and $password){
-            $webRequest.Credentials = New-Object System.Net.NetworkCredential($username, $password)
+                $webRequestHead.Credentials = New-Object System.Net.NetworkCredential($username, $password)
         }
+        $webRequestHead.Method = "HEAD"
+        [System.Net.HttpWebResponse]$webResponseHead = $webRequestHead.GetResponse()
+        $modified = $webResponseHead.LastModified
```

Comment:
I think maybe call $webResponseHead.Close() here so the code doesn't start a second request without closing the first connection.